# 9 ensures that blur is big enough to smooth the image enough to avoid 
# too many false circles 
blur_kernel_size : 9
# 2 is the value that gets us the actual circle in most cases
blur_sigma : 2
# this is the default value that is the most reliable in our case
hough_dp : 1
# use a value greater to the width and height of the circle to make sure it
# only detects one circle
hough_min_dist : 1000
# this value of threshold_1 makes sure that the actual circle is always 
# detected
hough_threshold_1 : 30
# this value is optimal to avoid false circles; increasing this limits the
# detection of the actual circle as well
hough_threshold_2 : 15
# expected minimum radius of the circle
#15
hough_min_radius: 5
# expected maximum radius of the circle
#30
hough_max_radius: 30
